Build of result.c on UNIX system failed (except LINUX and FreeBSD).
According to RFC3493 include of netdb.h is missing

It will be submitted to upstream.

--- openscap-1.3.7/src/XCCDF/result.c
+++ openscap-1.3.7/src/XCCDF/result.c
@@ -51,6 +51,7 @@ #include <arpa/inet.h>
 #include <sys/socket.h>
 #include <pwd.h>
 #include <sys/types.h>
+#include <netdb.h>
 #endif
 
 #if defined(OS_LINUX)
